src/dumch/parse.cljc

```diff
@@ -287,38 +305,47 @@
         (str/replace #"(\s*@.*\n)" "\n")
 
         ;; cleaning code from comments
-        (str/replace #"\/\*(\*(?!\/)|[^*])*\*\/" "")    ; /* ... */ 
-        (str/replace #"(\/\/).*" "")                    ; // ... 
-        (str/replace str-pattern 
-                     (transform (comp substitute-curly-quotes 
-                                      decode)))))) 
+        (str/replace #"/\*(\*(?!/)|[^*])*\*/" "")    ; /* ... */ 
+        (str/replace #"(//).*" "")                   ; // ... 
```

Comment:
I think javascript wont work without escaping 
![image](https://user-images.githubusercontent.com/14236531/172898506-d1f01112-5315-4003-a8f8-4a03b9adbf51.png)
https://regex101.com/

Reply:
It wouldn't compile in ClojureScript with the escaping, actually. See https://clojurians.slack.com/archives/C03S1L9DN/p1654705094721129

I think it does the right thing in both Clojure and ClojureScript now, but then again, we do have failing tests... I haven't figured those out yet.